_posts/2020-12-08-resteasy-reactive.adoc

```diff
@@ -174,6 +174,9 @@ The extensions that integrate with the existing quarkus-resteasy extension also
 
 The project has landed in the Quarkus master branch, so if you’re eager to try it out, you'll have to build Quarkus from source by following https://github.com/quarkusio/quarkus/blob/master/CONTRIBUTING.md#building-master[this] and use the proper BOM and version following https://github.com/quarkusio/quarkus/blob/master/CONTRIBUTING.md#updating-the-version[this].
 
+Furthermore, you can also use Maven Snapshots, by specifying version `999-SNAPSHOT` as the Quarkus version and using `quarkus-bom` instead of `quarkus-universe-bom` as the BOM.
+There are various was to enable Snapshot versions in Maven. https://stackoverflow.com/a/7717234/2504224[This] StackOverflow answer shows configuration that can be used either on a per project basis or globally.
```

Comment:
s/was/ways/ ?
s/Snapshot/snapshot/ ?

On a more general note, should't it state were the snapshots are deployed?

Reply:
Thanks for spotting the typos!

Good point about the snapshots, I added a note